spec/models/tag/hashtag_spec.cr

```diff
@@ -12,7 +12,7 @@ Spectator.describe Tag::Hashtag do
 
     it "associates with an actor" do
       expect{described_class.new(subject: actor, name: "actor").save}.to change{Tag::Hashtag.count}.by(1)
-      expect(described_class.find(name: "actor").subject).to eq(actor)
+      expect(described_class.find(name: "actor").subject.iri).to eq(actor.iri)
```

Comment:
if the column is defined to accept null (see my longer reply to the PR) then these changes won't be necessary.

Reply:
I suggest that checking whole object equality here is not a good idea in general.

Reply:
i'm open to the argument. semantically it should be fine, however, and equality comparisons at the object level are supported and meant to be correct.